custom-metrics-autoscaling/prometheus-to-sd/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# Copyright 2018 The Kubernetes Authors. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM gcr.io/google-containers/debian-base-amd64:0.1
+
+ADD prometheus_dummy_exporter prometheus_dummy_exporter
```

Comment:
this won't work on Mac because you're not using GOOS

Reply:
ideally we should do a `go get ./...` here, then build with `CGO_ENABLED=0` and use multi-stage builds to deliver the binary in an alpine image, since this is a Go binary.

Reply:
Done